src/main/java/com/google/sps/servlets/DishAnalysisServlet.java

```diff
@@ -40,25 +41,26 @@
 import javax.ws.rs.client.WebTarget;
 import javax.ws.rs.core.MediaType; 
 import com.google.gson.Gson;
+import java.util.Set;
+import java.util.HashSet;
+import java.util.Arrays;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.MultipartConfig;
 
-
-<<<<<<< HEAD:src/main/java/com/google/sps/servlets/DishAnalysisServlet.java
 /** Servlet that uses VisionAPI to analyze uploaded images */
+@MultipartConfig
 @WebServlet("/dishAnalysis")
 public class DishAnalysisServlet extends HttpServlet {
 
   @Override
-  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
```

Comment:
I really don't understand why those changes weren't saved from before. But yes this looks good!